scripts/build_energy_totals.py

```diff
@@ -449,58 +449,62 @@ def idees_per_country(ct: str, base_dir: str) -> pd.DataFrame:
     assert df.index[39] == "Battery electric vehicles"
     ct_totals["electricity other road passenger"] = df.iloc[39]
 
-    assert df.index[41] == "Light duty vehicles"
+    assert df.index[41] == "Light commercial vehicles"
     ct_totals["total light duty road freight"] = df.iloc[41]
 
     assert df.index[49] == "Battery electric vehicles"
     ct_totals["electricity light duty road freight"] = df.iloc[49]
 
-    row = "Heavy duty vehicles (Diesel oil incl. biofuels)"
+    row = "Heavy goods vehicles (Diesel oil incl. biofuels)"
     ct_totals["total heavy duty road freight"] = df.loc[row]
 
     assert df.index[61] == "Passenger cars"
     ct_totals["passenger car efficiency"] = df.iloc[61]
 
     df = pd.read_excel(fn_transport, "TrRail_ene", index_col=0)
 
-    ct_totals["total rail"] = df.loc["by fuel (EUROSTAT DATA)"]
+    ct_totals["total rail"] = df.loc["by fuel"]
 
     ct_totals["electricity rail"] = df.loc["Electricity"]
 
-    assert df.index[15] == "Passenger transport"
-    ct_totals["total rail passenger"] = df.iloc[15]
+    assert df.index[9] == "Passenger transport"
+    ct_totals["total rail passenger"] = df.iloc[9]
 
-    assert df.index[16] == "Metro and tram, urban light rail"
-    assert df.index[19] == "Electric"
-    assert df.index[20] == "High speed passenger trains"
-    ct_totals["electricity rail passenger"] = df.iloc[[16, 19, 20]].sum()
+    assert df.index[10] == "Metro and tram, urban light rail"
+    assert df.index[13] == "Electric"
+    assert df.index[14] == "High speed passenger trains"
+    ct_totals["electricity rail passenger"] = df.iloc[[10, 13, 14]].sum()
 
-    assert df.index[21] == "Freight transport"
-    ct_totals["total rail freight"] = df.iloc[21]
+    assert df.index[15] == "Freight transport"
+    ct_totals["total rail freight"] = df.iloc[15]
 
-    assert df.index[23] == "Electric"
-    ct_totals["electricity rail freight"] = df.iloc[23]
+    assert df.index[17] == "Electric"
+    ct_totals["electricity rail freight"] = df.iloc[17]
 
     df = pd.read_excel(fn_transport, "TrAvia_ene", index_col=0)
 
-    assert df.index[6] == "Passenger transport"
-    ct_totals["total aviation passenger"] = df.iloc[6]
+    assert df.index[4] == "Passenger transport"
+    ct_totals["total aviation passenger"] = df.iloc[4]
 
-    assert df.index[10] == "Freight transport"
-    ct_totals["total aviation freight"] = df.iloc[10]
+    assert df.index[8] == "Freight transport"
+    ct_totals["total aviation freight"] = df.iloc[8]
 
-    assert df.index[7] == "Domestic"
-    ct_totals["total domestic aviation passenger"] = df.iloc[7]
+    assert df.index[2] == "Domestic"
+    ct_totals["total domestic aviation passenger"] = df.iloc[2]
 
-    assert df.index[8] == "International - Intra-EU"
-    assert df.index[9] == "International - Extra-EU"
-    ct_totals["total international aviation passenger"] = df.iloc[[8, 9]].sum()
+    # TODO added Ukraine to intra EU flights
```

Comment:
double-check: UK is normally IDEES code for United Kingdom

Reply:
> double-check: UK is normally IDEES code for United Kingdom

Ohh right (=